frontend/packages/dev-console/src/components/formik-fields/ResourceLimitField.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import cx from 'classnames';
+import { useField, useFormikContext, FormikValues } from 'formik';
+import { FormGroup, ControlLabel, HelpBlock } from 'patternfly-react';
+import { RequestSizeInput } from '@console/internal/components/utils';
+import { getValidationState } from './field-utils';
+import { ResourceLimitFieldProps } from './field-types';
+
+const ResourceLimitField: React.FC<ResourceLimitFieldProps> = ({
```

Comment:
Not for this PR, but it would be good to someday make this a reusable component outside of dev console so we can add a compute resource editor for any workload.

Reply:
We have a [reusable implementation](https://github.com/openshift/console/blob/master/frontend/public/components/operator-lifecycle-manager/descriptors/spec/resource-requirements.tsx) for OLM Operators that could be merged with this. Another disadvantage of having so many teams working on this large codebase is that I completely missed this until now.